### DISCUSSION TITLE
value error while plotting

### DISCUSSION THREAD

**Discussion by satya0210** [jax-ml/jax#16545] [General]
after calculating the value count  by using group by   i got a data frame
class
tt     h         A         B         C  
1      1        50%     50%    0

24      24        25%   25%   50%

Now i want to plot them by using
fig,ax = plt.subplots(figsize=(20,8))

x=K['HOUR']
y1=K['A']
y2=K['B']
y3=K['C']
y4=K['D']
y5=K['E']
y6=K['F']
y7=K['G']

ax.plot(x,y1,color="red", marker="o", label='A',linewidth='4')

but it is showing error
ValueError: Multi-dimensional indexing (e.g. `obj[:, None]`) is no longer supported. Convert to a numpy array before indexing instead.
previously I used  same code to plot then it shows now it is not showing

**Comment by jakevdp** [comment]
This looks unrelated to JAX; I'd suggest posting your question on StackOverflow with the `pandas` and `matplotlib` tags.
